index.ts

```diff
@@ -208,6 +208,26 @@ const ithacanet_chain = new TezosChain(
   teztnetsHostedZone,
 )
 
+const ithacanet_signer = new TezosChain(
```

Comment:
It's a signer, not a chain. Here you probably need to create a new pulumi object `TezosSigner` and `TezosSignerParametersBuilder`, or just create a pulumi `Chart` directly.

Reply:
actually @harryttd I suppose the TezosSigner object really belongs into the npm module? Teztnets never switch to the npm module because of the special stuff happening in mondaynet/dailynet to fetch the build version, but there is no reason not to define `TezosSigner` inside the module and import it?